### DISCUSSION TITLE
PDF Extract Images (OCR) - is "On" for Activation or does it mean it is activated?

### DISCUSSION THREAD

**Discussion by gelbzucht** [open-webui/open-webui#1751] [Q&A]
I got confused, is OCR for Images activated when it shows "On" or is that to enable it? :) 
![image](https://github.com/open-webui/open-webui/assets/42704390/49dd31b6-60c7-4625-9a3f-25d4626b1dff)

**Comment by justinh-rahb** [comment]
OCR is active if it says `On`.

**Reply by gelbzucht** [reply]
thx!
